packages/react/src/components/Storage/FileUploader/FileUploader/FileUploader.tsx

```diff
@@ -7,13 +7,13 @@ import {
   Button as UploadButton,
   ComponentClassNames,
   VisuallyHidden,
-} from '../../../primitives';
+} from '../../../../primitives';
 
-import { useFileUploader } from './hooks/useFileUploader';
-import { UploadPreviewer } from './UploadPreviewer';
-import { UploadDropZone } from './UploadDropZone';
-import { UploadTracker } from './UploadTracker';
-import { FileState, FileUploaderProps } from './types';
+import { useFileUploader } from '../hooks/useFileUploader';
+import { UploadPreviewer } from '../UploadPreviewer';
+import { UploadDropZone } from '../UploadDropZone';
+import { UploadTracker } from '../UploadTracker';
+import { FileState, FileUploaderProps } from '../types';
```

Comment:
What do you think of moving the `FileUploaderProps` to a sibling `types.ts`?

Reply:
I am planning to do the types in a follow up PR once I have a better grasp of the codebase